pages/button/button.css

```diff
@@ -0,0 +1,87 @@
+.join {
+    border: none;
+    width: 80px;
+    padding: 0 16px;
+    line-height: 26px;
+    border-radius: 6px;
+    background-color: #1877f2;
+    font-size: 20px;
+}
+
+.sign {
+    border: none;
+    width: 80px;
+    padding: 0 16px;
+    line-height: 26px;
+    border-radius: 6px;
+    background-color: #00a400;
+    font-size: 20px;
+    color: #fff;
+}
+
+.learn {
+    border: none;
+    width: 130px;
+    padding: 0 16px;
+    line-height: 26px;
+    border-radius: 6px;
+    background-color: #2C3333;
+    font-size: 20px;
+}
+
+.newchat {
+    border: none;
+    width: 130px;
+    padding: 0 16px;
+    line-height: 24px;
+    border-radius: 6px;
+    background-color: #2C3333;
+    font-size: 14px;
+    color: white;
+   
+}
+
+.joined {
+    border: none;
+    width: 110px;
+    padding: 0 16px;
+    line-height: 26px;
+    border-radius: 8px;
+    background-color: #2C3333;
+    font-size: 16px;
+}
+
+.addfreind {
+    border: none;
+    width: 100px;
+    padding: 0 16px;
+    line-height: 26px;
+    border-radius: 8px;
+    font-size: 16px;
+    background-color: #283bb7;
+    cursor: pointer;
+}
+
+.drop {
+    width: 20px;
+    background-color: #2C3333;
+    padding: 0 16px;
+    border-radius: 3px;
+}
+
+.dropimage {
+    border-left: 1 rem solid blue;
+}
+
+h6 {
+    color: white;
+    text-align: center;
+    font-weight: bold;
+    text-align: top;
+    font-family: SFProDisplay-Bold, Helvetica, Arial, sans-serif !important;
```

Comment:
Don't use `!important`.

Please research on why we should not use `!important` and comment answer